Makefile

```diff
@@ -1,9 +1,9 @@
 DESTDIR =
 PREFIX  = /usr/local
 
-BASHDIR = /etc/bash_completion.d
+BASHDIR = $(PREFIX)/etc/bash_completion.d
 ZSHDIR  = /usr/share/zsh/vendor-completions
```

Comment:
These directories work in the Ubuntu image I use ([the Vagrantfile links them into place](https://github.com/ogham/exa/blob/d2b1499fb12c3fc57041f8eb4d82f615846e5286/Vagrantfile#L72-L79)), but that’s no guarantee that they’ll work in other places, too. None of them work on my computer running OS X.

I tried to find another project’s Makefile that installs shell completions, but came up short.